ingest/workflow/snakemake_rules/transform.smk

```diff
@@ -0,0 +1,131 @@
+"""
+This part of the workflow handles transforming the data into standardized
+formats and expects input file
+
+    sequences_ndjson = "data/sequences.ndjson"
+
+This will produce output files as
+
+    metadata = "data/metadata.tsv"
+    sequences = "data/sequences.fasta"
+
+Parameters are expected to be defined in `config.transform`.
+"""
+
+
+rule fetch_general_geolocation_rules:
+    output:
+        general_geolocation_rules="data/general-geolocation-rules.tsv",
+    params:
+        geolocation_rules_url=config["transform"]["geolocation_rules_url"],
+    shell:
+        """
+        # (1) Pick curl or wget based on availability    
+        if which curl > /dev/null; then
+            download_cmd="curl -fsSL --output"
+        elif which wget > /dev/null; then
+            download_cmd="wget -O"
+        else
+            echo "ERROR: Neither curl nor wget found. Please install one of them."
+            exit 1
+        fi
+
+        # (2) Fetch general geolocation rules
+        $download_cmd {output.general_geolocation_rules} {params.geolocation_rules_url}
```

Comment:
I would just use one or the other here. Both are available in all our runtimes. Many of our other workflows already assume `curl` is available, for example.

If we _really_ want to use either curl or wget based on availability, I'd wrap up this bit of shell into a separate program and invoke that, instead of embedding the code here in the rule.

But would really just use one (and probably curl).

I realize this was [spawned by commentary in another PR](https://github.com/nextstrain/ebola/pull/6#discussion_r1048835183), but I don't think it's necessary.